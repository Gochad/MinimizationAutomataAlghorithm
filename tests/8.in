A B C D
A
A D
0 1
A 0 C
A 1 B
B 0 D
B 1 A
C 0 A
C 1 D
D 0 B
D 1 C