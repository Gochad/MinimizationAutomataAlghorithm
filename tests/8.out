BC AD 
A
AD 
0 1 
BC 0 AD
BC 1 AD
AD 0 BC
AD 1 BC
